.github/workflows/test.yaml

```diff
@@ -0,0 +1,46 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  IP: 127.0.0.1
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
+      - name: create swarm cluster
+        run: ./contrib/create_swarm_cluster.sh
+      - name: deploy stack
+        run: ./contrib/deploy_stack.sh
+      - name: wait 15 seconds
+        run: sleep 15
+      - name: test faas swarm
+        run: make test-swarm
+        env:
+          OPENFAAS_URL: http://${{ env.IP }}:8080/
+      - name: clean swarm cluster
+        run: ./contrib/clean_swarm_cluster.sh
+      - name: test faas-netes
```

Comment:
followup to my previous comment, can we split the k8s setup too ?

Reply:
of course 👍

Reply:
done. I updated the screenshot in the comment too.